test/e2e/app-dir/app-edge/pages/pages-edge.tsx

```diff
@@ -2,4 +2,4 @@ export default function Page() {
   return <p>pages-edge-ssr</p>
 }
 
-export const config = { runtime: 'edge' }
+export const config = { runtime: 'experimental-edge' }
```

Comment:
@orinokai @sarahetter I put this back to `experimental-edge` as the build was complaining when I finally got this running locally. Maybe the Next version isn't up to date for the E2E tests we pull in that expects `runtime: 'edge'`? I'll take a peek at that and open an issue if that's the case.